src/components/inputs/CommentInput.tsx

```diff
@@ -88,6 +88,7 @@ export const CommentInput = ({ createComment, task_id }: Prop) => {
           placeholder="Leave a comment..."
           suggestions={assigneeSuggestions}
           editorClass="tapwrite-comment-input"
+          disablePasteAndDnd
```

Comment:
something for future, I would have probably named this prop disableImage. Under the hood the library knows that it needs to disable. In the future we might have another create method and in that case this would be less accurate. Also this prop is probably what should control the floating menu item being disabled.